### PR TITLE
Bind class methods in constructor

### PR DESCRIPTION
**Pull request by sethkrasnianski**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -176,44 +186,29 @@ class App extends Component {
...
+            name="route-select"
+            options={this.state.routes}
+            multi
+            onChange={this.handleRouteSelectChange}
+            value={this.state.selectedRoutes}
+          />
+        </div>
+        <RiptaMap
+          center={COORDS[this.state.direction].latLng}
+          zoom={COORDS[this.state.direction].zoon}
```
Comment (spinloop) [inline src/App.js]: typo here

Comment (spinloop) [inline src/App.js]: zoon

Comment (sethkrasnianski) [inline src/App.js]: good ol' zoon

**Comment by spinloop** [comment]
👍 🐐 :shipit:

**Pull request merged**